pyfolio/plotting.py

```diff
@@ -1660,3 +1662,126 @@ def plot_prob_profit_trade(round_trips, ax=None):
            xlim=(lower_plot, upper_plot), ylim=(0, y.max() + 1.))
 
     return ax
+
+
+def plot_cone(aggregate_returns, bounds, ax,
```

Comment:
If `plot_cone` is a private method that's only going to be called by `plot_multistrike_cones`, maybe we should declare it as 'private' by prefacing it with an `_`?